File: SpringBoot/src/main/java/com/scheduler/persistence/entity/Diary.java
package com.scheduler.persistence.entity;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter @Setter
public class Diary {
    private String id;
    private String user;
    private LocalDateTime dateTimeGenerated;
    private LocalDateTime dateTimeModified;
    private String text;
}

File: Angular/scheduler-ui/src/app/models/diary/diary.ts
export class Diary {
    id!: string;
    dateTimeGenerated!: Date;
    dateTimeModified!: Date;
    text!: string;
    displayDiaryDiv!: boolean;
    dateGeneratedFormat: string = "";
    timeGeneratedFormat: string = "";
    dateModifiedFormat: string = "";
    timeModifiedFormat: string = "";
}
